.gitlab-ci.yml

```diff
@@ -166,33 +166,35 @@ Base:
       - RUNNER:
           - aws/fedora-35-x86_64
           - aws/fedora-35-aarch64
-          - aws/rhel-8.5-ga-x86_64
-          - aws/rhel-8.5-ga-aarch64
+          - aws/rhel-8.6-ga-x86_64
+          - aws/rhel-8.6-ga-aarch64
+          - aws/rhel-9.0-ga-x86_64
+          - aws/rhel-9.0-ga-aarch64
           - aws/centos-stream-8-x86_64
           - aws/centos-stream-8-aarch64
       - RUNNER:
-          #- aws/rhel-8.6-nightly-x86_64
-          #- aws/rhel-8.6-nightly-aarch64
-          #- aws/rhel-9.0-nightly-x86_64
-          #- aws/rhel-9.0-nightly-aarch64
+          - aws/rhel-8.7-nightly-x86_64
+          - aws/rhel-8.7-nightly-aarch64
+          - aws/rhel-9.1-nightly-x86_64
+          - aws/rhel-9.1-nightly-aarch64
           - aws/centos-stream-9-x86_64
           - aws/centos-stream-9-aarch64
         INTERNAL_NETWORK: ["true"]
 
-#Manifests:
-#  stage: test
-#  extends: .terraform
-#  rules:
-#    - !reference [.upstream_rules, rules]
-#    - !reference [.nightly_rules, rules]
-#  script:
-#    - schutzbot/deploy.sh
-#    - /usr/libexec/tests/osbuild-composer/manifest_tests.sh
-#  parallel:
-#    matrix:
-#      - RUNNER:
-#          - aws/rhel-9.0-nightly-x86_64
-#        INTERNAL_NETWORK: ["true"]
+Manifests:
+  stage: test
+  extends: .terraform
+  rules:
+    - !reference [.upstream_rules, rules]
+    - !reference [.nightly_rules, rules]
+  script:
+    - schutzbot/deploy.sh
+    - /usr/libexec/tests/osbuild-composer/manifest_tests.sh
+  parallel:
+    matrix:
+      - RUNNER:
+          - aws/rhel-9.1-nightly-x86_64
```

Comment:
Lets use centos stream here to prevent us disabling it the next time, because the test does not care about the actual runner OS.